docs/content/license.md

```diff
@@ -5,6 +5,7 @@ anchor: "license"
 weight: 40
 ---
 
-This project is licensed under the [Apache 2.0](license) license. For the license of the used libraries you have to check the respective sources.
+This project is licensed under the [Apache 2.0](license) license.
+For the license of the used libraries you have to check the respective sources.
```

Comment:
I had put it into a single line on purpose, the new line also gets rendered by Hugo.